cmd/eksctl/get.go

```diff
@@ -43,10 +56,12 @@ func getClusterCmd() *cobra.Command {
 	fs := cmd.Flags()
 
 	fs.StringVarP(&cfg.ClusterName, "name", "n", "", "EKS cluster name")
+	fs.IntVar(&chunkSize, "chunk-size", DEFAULT_CHUNK_SIZE, "Return large lists in chunks rather than all at once. Pass 0 to disable.")
 
 	fs.StringVarP(&cfg.Region, "region", "r", DEFAULT_EKS_REGION, "AWS region")
 	fs.StringVarP(&cfg.Profile, "profile", "p", "", "AWS creditials profile to use (overrides the AWS_PROFILE environment variable)")
 
+	fs.StringVarP(&output, "output", "o", "log", "Specifies the output printer to use")
```

Comment:
I think `table` should be the default.

Reply:
I wasn't sure which one to make default so i went to "as-is" functionality. I'll change to table.